gta-gallery/src/services/axiosInstance.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+const baseURL = process.env.REACT_APP_API_URL;
+
+const defaultOptions = {
+  baseURL,
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
+
+const axiosInstance = axios.create(defaultOptions);
+
+axiosInstance.interceptors.request.use((config) => {
+  const token = localStorage.getItem('token');
```

Comment:
Check if there are token in local storage. If there are not then don't send Authorization Header